app/controllers/LandingController.scala

```diff
@@ -33,13 +33,12 @@ class LandingController @Inject() (
 ) extends FrontendController(mcc)
   with Logging {
 
-  val landingPage: Action[AnyContent] = as.landingPageAction { implicit request =>
-    logger.debug(s"inside landing page with journey: ${request.journey.toString}")
-    request.journey.origin match {
-      case Origins.Epaye.Bta         => Ok(views.epayeLanding())
-      case Origins.Epaye.DetachedUrl => Ok(views.epayeLanding())
-      case Origins.Epaye.GovUk       => Redirect(routes.DetermineTaxIdController.determineTaxId())
-    }
+  val landingPage: Action[AnyContent] = as.default { implicit request =>
```

Comment:
do we not want to keep the old logic?

Reply:
So at this point we don't have access to the origin - I've changed it to a default action - we may put it in somewhere else, but for now, it's not required, I don't think